draft-ietf-rtcweb-jsep.xml

```diff
@@ -1799,9 +1799,27 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>For media m= sections, JSEP implementations MUST support
           the "UDP/TLS/RTP/SAVPF" profile specified in
-          <xref target="RFC5764"></xref>, and MUST indicate this
-          profile for each media m= line they produce in an offer. For
-          data m= sections, implementations MUST support the
+          <xref target="RFC5764"></xref>. Implementations MUST indicate this
+          profile for each media m= line they produce in an offer unless the
+          media section contains only TCP candidates; if all candidates use TCP as a
+          transport, implementations MUST indicate "TCP/DTLS/RTP/SAVPF," to
+          allow for compatibility with non-WebRTC implementations of
+          <xref target="I-D.ietf-mmusic-ice-sip-sdp"></xref>.</t>
+          
+            <t><list style="hanging">
+              <t hangText="Note:">Current implementations of JSEP do not update m= or c= lines
+              after a candidate pair is selected.  When these facilities are
+              added, implementations should use TCP/DTLS/RTP/SAVPF as a
+              transport token when the candidate in question is using TCP, in
+              order to facilitate interactions with non-JSEP SIP/SDP
+              implementations, which assign meaning to the protcol field.
```

Comment:
```suggestion
              implementations, which assign meaning to the protocol field.
```

Reply:
"Current implementations" is tricky, because "current" will be whenever someone reads the spec - today or in 2 years - and what is claimed may change over time. Therefore, I suggest using "At the time of writing this specification", or something similar.

Reply:
What does "Current implementations of JSEP" refer to? Browsers implementing the API, or applications using the API? I think we shall make a clear separation, and say that the JSEP API, nor the current applications using it, update the m= or c= lines. I also think that we should recommend that applications that are expected to interact with legacy/non-JSEP endpoints to actually update the c= and m= lines.